pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "autopkg"
+authors = []
+version = "2.7.3"
+description = "AutoPkg is an automation framework for macOS software packaging and distribution"
+readme = "Readme.md"
+requires-python = ">=3.10"
+license = {text = "BSD-3-Clause"}
+classifiers = [
+    "Topic :: System :: Software Distribution",
+]
+dynamic = [ "dependencies" ]
+
+[project.scripts]
+autopkg = "autopkgcli.autopkg:main"
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["gh_actions_requirements.txt"]}
```

Comment:
I agree with this, the other dependency files have too much stuff in them.

Reply:
There is a lot in the current requirements that could be qualified as dev tools and not actually needed for the running of app. This should be trimmed.

Reply:
I do agree, it makes sense to make a distinction between requirements for run/build vs requirements for dev/test/validation. I do think there is too much stuff in some of the requirements files for the sake of doing the basics in a github action or just running recipes on a build server.

I think the reason for the additional items in the requirements is because the requirements were traditionally only ever used by people doing dev work, while the production way to use autopkg was to install the PKG file on MacOS.